.build/build-resolver.xml

```diff
@@ -134,6 +134,16 @@
             </dependencies>
             <path refid="checkstyle.classpath" classpath="runtime"/>
         </resolve>
+        <resolve>
+<!-- Required for running JavaScript regardless of JDK version, this is only a build dependency, not inlcuded in the distribution -->
```

Comment:
@jacek-lewandowski could you aligned this? just above `<dependencies>`

Reply:
will do